sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
More info on how this works to enforce postponed type evaluation

https://adamj.eu/tech/2021/05/15/python-type-hints-future-annotations/